Formula/cloudprober.rb

```diff
@@ -0,0 +1,40 @@
+class Cloudprober < Formula
+  desc "Active monitoring software to detect failures before your customers do"
+  homepage "https://cloudprober.org"
+  url "https://github.com/cloudprober/cloudprober/archive/refs/tags/v0.12.0.tar.gz"
+  sha256 "e2289eaad263e35e20ed3afaccf4c854df179de4e36aa705fcf106d93a352b0b"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "cloudprober", "VERSION=v#{version}"
+    bin.install "cloudprober"
```

Comment:
Is there a `make install` we can use?

Reply:
There is, it's just running `go install` under the hood.

Reply:
Yeh, let's use that then for future-proofing if they install more in future.

Reply:
I'd argue this is better. make install doesn't take the prefix, while bin.install will do the right thing for brew. No?

Reply:
`make install` should be able to take a prefix and is more future proof if in future you ever decide to install e.g. docs.

I didn't realise you were the upstream for this? We frown upon projects submitting themselves to Homebrew unless they can e.g. point to user requests for this. Do you have any issues etc. for this?

Reply:
> I didn't realise you were the upstream for this? We frown upon projects submitting themselves to Homebrew unless they can e.g. point to user requests for this. Do you have any issues etc. for this?

Actually, I am also the user here. I use brew myself and thought it will be easier for me (and Cloudprober users) if brew could install cloudprober. Also my another open-source project, pacparser, was already on brew, so I thought to give it a try.

I see your point regarding users requesting for it, but as a user I don't file a bug if a software is not available on brew. I just find it somewhere else. So by that criteria, you'll be missing a lot of genuine users.

In case, you're concerned about cloudprober not being worthy, I think Cloudprober has a significant user base (2.5M docker pulls, 1.4k stars on [the original project](https://github.com/google/cloudprober) (I had to move the development after I left the project) and users will benefit from it being on brew.

Reply:
Thanks for explaining @manugarg! Talked with some other maintainers and I'm 👍🏻 on including this.